google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/BigQueryImpl.java

```diff
@@ -560,15 +561,15 @@ public Boolean call() {
   }
 
   @Override
-  public Job query(QueryJobConfiguration configuration, JobOption... options)
-      throws InterruptedException {
+  public QueryResult query(QueryJobConfiguration configuration, JobOption... options)
+      throws InterruptedException, JobException {
```

Comment:
Does `JobException` need to move up to the interface, too?

Reply:
Good catch. Done